matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/Currency.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019 Triplebet Limited. All right reserved. Inchalla, Le Val, Alderney, GY9 3UL.
+ * Company Registration Number: 1827.
+ */
+
+package com.matchbook.sdk.core.clients.rest.dtos.prices;
+
+public enum Currency {
+
+    EUR, GBP, USD
```

Comment:
are you sure MB support only those currencies?

Reply:
No, there are more. Though, as other options, I'm keeping the API as much simple as possible, at least initially. Even if it is probably safe to add all currencies from the beginning.